pyanaconda/startup_utils.py

```diff
@@ -433,6 +462,8 @@ def parse_kickstart(options, addon_paths):
         log.info("Parsing kickstart: " + ks)
         ksdata = kickstart.parseKickstart(ks, options.ksstrict)
 
+        if "run_boss" in flags.cmdline:
+            distribute_kickstart_with_boss(ks)
```

Comment:
I'm not sure if there is better place but this is side effect and it will be hard to find how the kickstart is distributed when it's here. I would rather see this call somewhere else.

Reply:
I think we need either to split the parse_kickstart function or do passing to boss in this function. I'd clean this up (parse_kickstart) when we know better what we want to do (for example parse only modules-unprocessed part in anaconda?). For now, we just want to be able to pass kickstart to modules to be able to work on their POCs/integration.
What about - to be more explicit - adding pass_to_boss flag to the parse_kickstart function? I'll add a patch doing that to the PR.